x/data/server/features/anchor.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Anchor
+
+  Scenario: data is anchored when the data has not been anchored
+    Given a valid content hash
```

Comment:
can we have an example, maybe in proto json in a gherkin doc string

Reply:
duplicate